examples/jsm/renderers/webgpu/WebGPUBackground.js

```diff
@@ -1,7 +1,7 @@
 import { GPULoadOp, GPUStoreOp } from './constants.js';
 import { Color, Mesh, BoxGeometry, BackSide } from 'three';
-import { context, transformDirection, positionWorld, modelWorldMatrix } from 'three-nodes/Nodes.js';
-import MeshBasicNodeMaterial from 'three-nodes/materials/MeshBasicNodeMaterial.js';
+import { context, transformDirection, positionWorld, modelWorldMatrix } from 'three/nodes';
+import { MeshBasicNodeMaterial } from 'three/nodes';
```

Comment:
I think these two imports can be merged into one.

Reply:
Thanks, good catch!